package.json

```diff
@@ -38,6 +38,8 @@
     }
   },
   "dependencies": {
+    "@ensdomains/ens-contracts": "0.0.7",
+    "@ensdomains/ensjs": "^2.0.1",
```

Comment:
are we having package-lock/nvm version problems again? package-lock diff is 41k chagnes

Reply:
yea make sure you're running npm v1. you changed the whole structure of the lockfile with this cause you have npm v2. make sure you run `nvm use` so you're using the right node version, and then `npm install npm@1` ... then do `git checkout master -- package.json package-lock.json && npm install --save @ensdomains/ens-contracts @ensdomains/ensjs`

Reply:
Have you done this yet?